roles/edpm_neutron_ovn/defaults/main.yml

```diff
@@ -0,0 +1,48 @@
+---
+# defaults file for edpm_ovn
+
+# seconds between retries for download tasks
+edpm_neutron_ovn_images_download_delay: 5
+
+# number of retries for download tasks
+edpm_neutron_ovn_images_download_retries: 5
+
+edpm_neutron_ovn_config_src: /var/lib/openstack/configs/neutron-ovn
+edpm_neutron_ovn_agent_config_dir: /var/lib/config-data/ansible-generated/neutron-ovn-agent
+edpm_neutron_ovn_agent_log_dir: "/var/log/neutron"
+
+edpm_neutron_ovn_agent_image: "quay.io/podified-antelope-centos9/openstack-neutron-ovn-agent:current-podified"
+
+edpm_neutron_ovn_common_volumes:
+  - /run/openvswitch:/run/openvswitch:z
+  - "{{ edpm_neutron_ovn_agent_config_dir }}:/etc/neutron.conf.d:z"
+  - /run/netns:/run/netns:shared
```

Comment:
I don't think it spawns any processes, does it? Why do we need netns?

Reply:
Yes right, as of now it doesn't spawn, will remove it.